data/abilities.js

```diff
@@ -536,8 +536,8 @@ exports.BattleAbilities = {
 		num: 56,
 	},
 	"damp": {
-		desc: "While this Pokemon is active, Self-Destruct, Explosion, and the Ability Aftermath are prevented from having an effect.",
-		shortDesc: "While this Pokemon is active, Self-Destruct, Explosion, and Aftermath have no effect.",
+		desc: "While this Pokemon is active, Explosion, Mind Blown, Self-Destruct, and the Ability Aftermath are prevented from having an effect.",
+		shortDesc: "Prevents Explosion/Mind Blown/Self-Destruct/Aftermath while this Pokemon is active.",
```

Comment:
Hmm I wonder if new users who read this will think “Prevents the move from executing.” This might be a bit vague.

Reply:
Although I don’t have a better idea with the little wording space you can have

Reply:
does "Pokemon are immune to Explosion/Mind Blown/Self-Destruct/Aftermath while this Pokemon is active." fit?

"While active: Pokemon are immune to Explosion/Mind Blown/Self-Destruct/Aftermath." could work too, idk just some ideas

Reply:
Apparently damp prevents the use of the moves not just preventing damage so this is fine actually

Reply:
ok.